<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindow</class>
 <widget class="QMainWindow" name="ProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/appicon</normaloff>:/icons/appicon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QtCharts::QChartView" name="chartView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowLabels"/>
   <addaction name="actionShowIndividual"/>
   <addaction name="actionShowAverage"/>
   <addaction name="actionSavePlot"/>
  </widget>
  <action name="actionShowAverage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/show-average</normaloff>:/icons/show-average</iconset>
   </property>
   <property name="text">
    <string>Average Profile</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionShowIndividual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/show-individual</normaloff>:/icons/show-individual</iconset>
   </property>
   <property name="text">
    <string>Individual Profiles</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionSavePlot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save Plot</string>
   </property>
   <property name="shortcut">
    <string>Print</string>
   </property>
  </action>
  <action name="actionShowLabels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/show-labels</normaloff>:/icons/show-labels</iconset>
   </property>
   <property name="text">
    <string>Labels</string>
   </property>
   <property name="toolTip">
    <string>Show Sample Labels</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
